test_dpath.py

```diff
@@ -0,0 +1,58 @@
+import dpath.util
```

Comment:
@benoit-cty Is this file here to test that dpath works as expected on conda install or is it a residue from another subject? It's in the root directory and I don't remember seeing it on this PR on first review. Remove it? 🤔

Reply:
Nice catch ! I will remove it.